drivers/storage/portworx/component/portworx_basic.go

```diff
@@ -262,9 +262,14 @@ func (c *portworxBasic) createClusterRole() error {
 				},
 				{
 					APIGroups: []string{"storage.k8s.io"},
-					Resources: []string{"storageclasses"},
+					Resources: []string{"storageclasses", "csinodes", "volumeattachments"},
```

Comment:
`volumeattachments` is redudant. Also present on line 270.

Reply:
ah, didn't remove one, fixing